src/main/java/seedu/ezdo/ui/StatusBarFooter.java

```diff
@@ -61,7 +61,7 @@ private void setSyncStatus(String status) {
 
     @Subscribe
     public void handleEzDoChangedEvent(EzDoChangedEvent ezce) {
-        String lastUpdated = (new SimpleDateFormat("dd/MM/YYYY (E) h:mm:ss a")).format(new Date());
+        String lastUpdated = (new SimpleDateFormat("dd/MM/YYYY H:mm:ss")).format(new Date());
```

Comment:
Might want to move magic string to a constant 😄  Tutor commented on mine about this.